#! /bin/sh /usr/share/dpatch/dpatch-run
## 45_misc_doc_changes.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Misc doc changes

@DPATCH@
diff -urNad unstable~/README_FILES/README.courier unstable/README_FILES/README.courier
--- unstable~/README_FILES/README.courier	2006-11-04 00:42:23.000000000 -0300
+++ unstable/README_FILES/README.courier	2006-11-04 00:42:44.192140909 -0300
@@ -1,3 +1,7 @@
+Debian Note:  Rename the start links of amavisd-new to S21, or you
+risk trouble. See the last paragraph of this text for more information.
+
+
 How to use amavisd-new with Courier
 ***********************************
 
diff -urNad unstable~/README_FILES/README.exim_v3 unstable/README_FILES/README.exim_v3
--- unstable~/README_FILES/README.exim_v3	2006-11-04 00:42:23.000000000 -0300
+++ unstable/README_FILES/README.exim_v3	2006-11-04 00:42:44.192140909 -0300
@@ -1,3 +1,9 @@
+Debian notes:
+
+1. Install amavisd-new-milter. It contains /usr/sbin/amavis,
+which is required for the following instructions to work.
+--- cut ---
+
 +======================================================================+
 | (Please see instructions in README.exim_v4. The setup described here |
 | is not recommended with amavisd-new + Exim v4)                       |
diff -urNad unstable~/README_FILES/README.milter unstable/README_FILES/README.milter
--- unstable~/README_FILES/README.milter	2006-11-04 00:42:23.000000000 -0300
+++ unstable/README_FILES/README.milter	2006-11-04 00:42:44.193140788 -0300
@@ -1,3 +1,24 @@
+Debian notes:
+
+1. Install amavisd-new-milter.
+
+2.  The postinst script needs to add lines to one's sendmail.mc file that
+tell it how to communicate with a milter program, e.g.:
+
+define(`MILTER',1)
+INPUT_MAIL_FILTER(`/usr/sbin/amavis-milter',`S=local:/var/lib/amavis/amavisd-new-milter.sock, F=T, T=S:10m;R:10m;E:10m')
+
+Run sendmailconfig.
+
+3.  Change/replace the following settings in /etc/amavis/amavisd.conf:
+
+$unix_socketname = "/var/lib/amavis/amavisd.sock";
+$forward_method = undef;
+$notify_method = $notify_method = 'pipe:flags=q argv=/usr/sbin/sendmail -i -odd -f ${sender} -- ${recipient}';
+
+Contributed and adapted from Susan Kleinmann <sgk@kleinmann.com>.
+[End of Debian Notes]
+
 ===============================================================================
 NOTE:
   A recommended sendmail setup is described in file README.sendmail-dual,
diff -urNad unstable~/amavisd.conf-default unstable/amavisd.conf-default
--- unstable~/amavisd.conf-default	2006-11-04 00:42:23.000000000 -0300
+++ unstable/amavisd.conf-default	2006-11-04 00:42:44.191141030 -0300
@@ -1,5 +1,8 @@
 use strict;
 
+# ADMINSTRATORS: DO NOT CHANGE THIS FILE
+# Change /etc/amavis/conf.d/30-user instead.
+
 ## A CONFIGURATION FILE FOR AMAVISD-NEW, LISTING ALL CONFIGURATION VARIABLES
 ## WITH THEIR DEFAULT VALUES (FOR REFERENCE ONLY, NON-AUTHORITATIVE)
 
